app/lib/constants.js

```diff
@@ -20,12 +18,14 @@ module.exports = {
       site: '@nhsuk',
     },
   },
+  assetsUrl: 'https://assets.nhs.uk',
```

Comment:
Is this a good thing to hard code? the assets in staging have different url

Reply:
Possibly not. It might be better to use an env var, however, it hasn't changed in this PR. And if it did changed based on the env var and that was different in the pre-live environment you have a situation where the first time the actual assets are being tested is when the site is running in live. Which isn't the first place I want to know if something is working!